molecule/ccloud/converge.yml

```diff
@@ -1,5 +1,5 @@
 ---
-- name: Remove the zookeeper and Kafka Broker Groups from Inventory
+- name: Remove the zookeeper, kafka controller and Kafka Broker Groups from Inventory
```

Comment:
Looks like below we are also removing kafka_controller: group also from the inventory. Line # 14-20

Reply:
yes, I thought that was the purpose since we were already removing zookeeper and broker. Should we not remove controller?

Reply:
make sense.